### PR TITLE
[jsk_2016_01_baxter_apc] Simplified version of #1536: Use object_data to publish work_order with APC2016 json

### PR DESCRIPTION
**Pull request by wkentaro**
Simplified #1536 

 - Remove apc2016 flag, and use object_data.
 - Get rosparams only in main func.
 - import jsk_apc2016_common
 - Remove no need max_weight currently.
 - ~is_apc2016 True as default.

### CODE REVIEW

File: jsk_2016_01_baxter_apc/node_scripts/work_order.py [hunk 1]
```diff
@@ -46,37 +62,62 @@ def get_work_order_msg(json_file):
...
+    is_apc2016 = rospy.get_param('~is_apc2016', False)
     if json_file is None:
         rospy.logerr('must set json file path to ~json')
         return
+    object_data = None
+    if is_apc2016:
+        object_data = jsk_apc2016_common.get_object_data()
 
-    msg = get_work_order_msg(json_file)
+    msg = get_work_order_msg(json_file, gripper, object_data)
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/node_scripts/work_order.py]: when you use apc2015, it gets error

Comment (knorth55) [inline jsk_2016_01_baxter_apc/node_scripts/work_order.py]: anyway, if it does not get error it is same as using apc2016 flag

File: jsk_2016_01_baxter_apc/node_scripts/work_order.py [hunk 2]
```diff
@@ -46,37 +62,62 @@ def get_work_order_msg(json_file):
...
         else:
+            # Without object_data, use defined blacklist for picking decision
             if target_object in abandon_target_objects:
+                # abandon if it is the target in the bin
+                jsk_logwarn(
+                    'Skipping {obj} in {bin} because it is listed to abandon'
+                    .format(obj=target_object_data['name'], bin=bin_))
                 continue
-            if [bin_object for bin_object in bin_contents[bin_] if bin_object in abandon_bin_objects]:
+            if any(obj in bin_contents[bin_] for obj in abandon_bin_objects):
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/node_scripts/work_order.py]: why do you prefer any?

**Comment by wkentaro** [comment]
closed via #1536

**Pull request closed without merging**